apps/server/src/modules/room/api/room.controller.ts

```diff
@@ -278,4 +280,23 @@ export class RoomController {
 
 		return Promise.resolve(response);
 	}
+
+	@Post(':roomId/duplicate')
+	@ApiOperation({
+		summary: ' creates a copy of the given room. Can only be used if you are the owner or admin of the origin room',
+	})
+	@ApiResponse({ status: HttpStatus.OK, description: 'successfully duplicated', type: String })
+	@ApiResponse({ status: HttpStatus.BAD_REQUEST, type: ApiValidationError })
+	@ApiResponse({ status: HttpStatus.UNAUTHORIZED, type: UnauthorizedException })
+	@ApiResponse({ status: HttpStatus.FORBIDDEN, type: ForbiddenException })
+	@ApiResponse({ status: '5XX', type: ErrorResponse })
+	@RequestTimeout('INCOMING_REQUEST_TIMEOUT_COPY_API')
+	public async duplicateRoom(
+		@CurrentUser() currentUser: ICurrentUser,
+		@Param() urlParams: RoomUrlParams
+	): Promise<RoomIdResponse> {
```

Comment:
```suggestion
	): Promise<RoomItemResponse> {
```

Reply:
Maybe we should even use the `CopyStatus` like in all other copy (duplicate) operations.